src/DynamoCoreWpf/Manipulation/DynamoManipulatorContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Dynamo.Controls;
+
+namespace Dynamo.Wpf.Manipulation
+{
+    public class DynamoManipulatorContext
+    {
+        internal DynamoManipulatorContext() { }
+
+        public DynamoView View { get; set; }
```

Comment:
We're not *totally* sure that the `DynamoView` is the only context in which a manipulator would be created. I can imagine other contexts.